linux_os/guide/services/ssh/file_permissions_sshd_private_key/rule.yml

```diff
@@ -16,10 +21,11 @@ identifiers:
 references:
     cui: 3.1.13,3.13.10
     disa: "366"
-    nist: AC-6,AC-17
+    nist: AC-6,AC-17,CM-6(b)
```

Comment:
CM-6(b) is an organization control, stating orgs must implement the controls created in CM-6(a). Individual configuration checks should not map to CM-6(b).

ref: https://nvd.nist.gov/800-53/Rev4/control/CM-6